examples/evolve_dstream_surf.py

```diff
@@ -288,13 +329,18 @@ def _get_df_save_handler(path: str) -> typing.Callable:
     # do simulation
     common_ancestor = Organism.create_founder()
     init_population = [common_ancestor.CreateOffspring() for _ in range(100)]
-    end_population = evolve_drift(init_population)
+    end_population = evolve_drift(
```

Comment:
We should probably rename the end population variable to something like sampled genomes